### DISCUSSION TITLE
Jiggle is different at high framerates. Any way to fix?

### DISCUSSION THREAD

**Discussion by KoKoEpic** [detomon/wigglebone#14] [Q&A] [answered]
I am using a 165hz monitor and noticed the jiggle isn't that good at high frame rates but looks great at 60 FPS. Is there any possible way to fix this? Sorry if this is a dumb question. Any help would be greatly appreciated! :)

**Comment by detomon** [comment, marked as answer]
You are right, this is a problem with the current implementation. I have been working on a more robust solution that is independent of the frame rate. I hope to release it in the next few days.

**Reply by KoKoEpic** [reply]
Great to hear! Thank you so much for replying and making a great addon! I am very much looking forward to the update and will try it out as soon as it's ready :)

**Reply by detomon** [reply]
The latest release adds new nodes which should not have any frame rate issues (see [release notes](https://github.com/detomon/wigglebone/discussions/15)). The new version requires Godot 4.3 but a fallback implementation for Godot 4.2 can be found in the [godot-4.2](https://github.com/detomon/wigglebone/tree/godot-4.2) branch.

**Reply by KoKoEpic** [reply]
I just tried it out and it works perfectly at high frame rates! Thanks so much! Very impressive work :)
